resources/definitions/bel_all_metal_delta.def.json

```
@@ -0,0 +1,94 @@
+{
+    "version":      2,
+    "name":         "Blue Eagle Labs All Metal Delta",
+    "inherits":     "fdmprinter",
+
+    "metadata": {
+        "author":                          "Johan Pettersson",
+        "visible":                         true,
+        "manufacturer":                    "Blue Eagle Labs",
+		"platform":                		   "bel_all_metal_delta.stl",
+        "file_formats":                    "text/x-gcode",
+        "has_machine_quality":             true,
+        "has_materials":                   true,
+        "has_variants":                    false,
+        "variants_name":                   "Head",
+        
+        "preferred_quality_type":          "a",
+        "preferred_material":              "generic_pla",
+        
+        "machine_extruder_trains": {
+            "0": "bel_all_metal_delta_extruder_0"
+        }
+    },
+
+    "overrides": {
+		"machine_name":            		   { "default_value": "All Metal Delta" },
+        "machine_width":           		   { "default_value": 280 },
+        "machine_depth":           		   { "default_value": 280 },
+        "machine_height":          		   { "default_value": 200 },
```

According to https://www.youtube.com/watch?v=X5UeCGArVt4 and https://www.kickstarter.com/projects/blueeaglelabs/the-metal-delta-build-your-own-metal-delta-printer, the build volume size is 250x250x280mm. Did you deviate from this size for a reason?
```suggestion
        "machine_width":           		   { "default_value": 250 },
        "machine_depth":           		   { "default_value": 250 },
        "machine_height":          		   { "default_value": 280 },
```